webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/DataVault.scala

```diff
@@ -5,6 +5,9 @@ import com.scalableminds.util.tools.Fox
 import scala.concurrent.ExecutionContext
 
 trait DataVault {
+  val MAX_EXPLORED_ITEMS_PER_LEVEL = 10
```

Comment:
I’d say this is not a property of DataVault, but of the exploring code. Maybe move this to the abstract explorer or ExploreRemoteLayerService or something like that? DataVault should not have to know that it is used to explore datasets.